### PR TITLE
Add slab metric to bytes_used memory metric

### PR DESCRIPTION
**Pull request by dhrupadb**

### CODE REVIEW

**Review by bmoyles0117 — COMMENTED** [review]
LGTM

**Review by bmoyles0117 — APPROVED** [review]
LGTM

**Review by qingling128 — APPROVED** [review]
LGTM.

File: src/memory.c [hunk 1]
```diff
@@ -356,12 +356,14 @@ static int memory_read_internal (value_list_t *vl)
 		MEMORY_SUBMIT ("used",        mem_used,
 		               "buffered",    mem_buffered,
 		               "cached",      mem_cached,
-		               "free",        mem_free);
+		               "free",        mem_free,
+		               "slab",        mem_slab_total);
```
Comment (igorpeshansky) [inline src/memory.c]: Let's add "slab" before "free"... Ditto below.

**Review by igorpeshansky — CHANGES_REQUESTED** [review]
One minor thing, otherwise LGTM.

**Review by dhrupadb — COMMENTED** [review]
PTAL

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Comment by igorpeshansky** [comment]
dhrupadb was at Google at the time.

**Pull request merged**